### ISSUE TITLE
Сannot authorize the user by sessionoid

### ISSUE THREAD

**Issue by Negatifff** [coova/coova-chilli#563]
Hello.
I have been using version 1.4 for a very long time and of course I have a certain number of scripts that help me manage it. one of them - user authorization by sessionid: **chilli_query login sessionid xxxxxxxxxxxxxx username xxxx password xxxx**

but in versions 1.5 and 1.6, user authorization by sessionid does not work. By mac-address and by ip it works as it should, but nothing happens to the sessionid - even the radius request is not sent.
It seems to me that this is some kind of bug or something has changed since version 1.5, about which I did not find information.

**Comment by sevan** [comment]
if you're comfortable building your own version of coova, try git bisect to isolate which commit broke things for you.

**Comment by Negatifff** [comment]
If only I knew which commit led to this behavior...

**Comment by sevan** [comment]
right, which [git bisect](https://git-scm.com/docs/git-bisect) will help you do, you have a starting point where it is worked (where 1.4 was tagged), it will help you to pinpoint which is the offending commit(s) between 1.4 and head.

**Comment by predators46** [comment]
maybe you want to try version 1.3.0
